src/test/java/org/mifos/integrationtest/cucumber/PaybillStepDef.java

```diff
@@ -0,0 +1,22 @@
+package org.mifos.integrationtest.cucumber;
+
+import io.cucumber.core.internal.com.fasterxml.jackson.databind.ObjectMapper;
+import org.mifos.integrationtest.config.PaybillConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class PaybillStepDef {
+    public String amsName;
+    @Autowired
+    PaybillConfig paybillConfig;
+    @Autowired
+    ObjectMapper objectMapper;
+    String paybillBpmnName;
+    String response;
+    Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    public String getAbsoluteFilePathToResource() {
```

Comment:
the generic version of this method is already implemented in  `Utils.java` file in package `org.mifos.integrationtest.common`, please use that.

Reply:
It is not there.

Reply:
that PR is not merged my manoj. @fynmanoj can you plzz merge my PR?? 
https://github.com/openMF/ph-ee-connector-integration-test/pull/3

Reply:
ok